source/install/WSL.rst

```diff
@@ -0,0 +1,25 @@
+Setting up Windows to use QIIME 2
+---------------------------------
+
+This section will demonstrate how to set up windows to allow QIIME 2 installation. Windows now allows users to install linux as a windows subsystem (WSL), the simplest way to use QIIME 2 on a machine with Windows OS is therefore to use this Windows subsystem for Linux.
```

Comment:
Unfortunately I don't think WSL is available to _many_ Windows users, so maybe we should temper the language used here a little bit.

```suggestion
This tutorial demonstrates how to set up QIIME 2 on the Windows Subsystem for Linux.
```